src/parser.ts

```diff
@@ -617,6 +639,19 @@ function parseFilesFromToolArguments(args: string, exts: string[]): string[] {
     match = regexp.exec(args);
     while (match) {
         let result: string = match[1];
+
+        // It is quite common to encounter the following pattern:
+        //  `test -f 'sourceFile.c' || echo './'`sourceFile.c
+        // Until we implement the correct approach (to query live the test command)
+        // we can just ignore it and consider the second option of the OR
+        // (by removing './'`). Even if this is hacky, it is worth it
```

Comment:
ok. can we track this with an issue?

Reply:
https://github.com/microsoft/vscode-makefile-tools/issues/81